mk/config.mk

```diff
@@ -943,6 +943,12 @@ ifeq (y-y,$(CFG_WITH_PAGER)-$(CFG_MEMTAG))
 $(error CFG_WITH_PAGER and CFG_MEMTAG are not compatible)
 endif
 
+# Privileged Access Never (PAN, part of the ARMv8.1 Extensiosn) can be
+# used to restrict accesses to unprivileged memory from privileged mode.
+CFG_PAN ?= n
```

Comment:
PAN is arm specific. Rename to `CFG_ARM_PAN`?

Reply:
I chose config name after `CFG_MEMTAG`, which is probably only available on ARM. That being said, if you think `CFG_ARM_PAN` if more appropriate, I will rename the config

Reply:
@jenswi-linaro, @jforissier: `CFG_PAN` or `CFG_ARM_PAN`? or maybe generic `CFG_PRIVILEGED_ACCESS_NEVER` (that Arm archs can implement using PAN CPU feature).

Reply:
`CFG_PAN` sounds good to me, PAN is PAN and short names are better. In addition, the macro is used only in Arm-specific files so there is no need to make it sound more general than what it means.

Reply:
PAN is PAN? IDTS: https://en.wikipedia.org/wiki/Pan
That said, I have no strong opinion on this by that i don't really like abusive use of acronyms that can be confusing IMHO, FWIW.

Reply:
I suggest to rephrase "PAN, part of the ARMv8.1 Extensiosn" to "PAN in part of the ARMv8.1 Extensiosn" since `CFG_PAN` is not Arm architecture specific.